src/test/java/solutions/fairdata/fdp/index/acceptance/api/entries/EntriesAll_GET.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * The MIT License
+ * Copyright © 2020 https://fairdata.solutions
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package solutions.fairdata.fdp.index.acceptance.api.entries;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import solutions.fairdata.fdp.index.WebIntegrationTest;
+import solutions.fairdata.fdp.index.api.dto.IndexEntryDTO;
+import solutions.fairdata.fdp.index.database.repository.EntryRepository;
+import solutions.fairdata.fdp.index.entity.IndexEntry;
+import solutions.fairdata.fdp.index.fixtures.IndexEntryFixtures;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+public class EntriesAll_GET extends WebIntegrationTest {
+
+    @Autowired
+    private EntryRepository entryRepository;
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    private URI url() {
+        return URI.create("/entries/all");
+    }
+
+    @Test
+    @DisplayName("HTTP 200: list empty")
+    public void res200_listEmpty() {
+        // GIVEN (prepare data)
+        mongoTemplate.getDb().drop();
+
+        // AND (prepare request)
+        RequestEntity<?> request = RequestEntity
+                .get(url())
+                .accept(MediaType.APPLICATION_JSON)
+                .build();
+        ParameterizedTypeReference<List<IndexEntryDTO>> responseType = new ParameterizedTypeReference<>() {
+        };
```

Comment:
This is repeated a couple of times throughout the tests, could this be made a constant?